node-zerox/src/index.ts

```diff
@@ -21,6 +21,9 @@ export const zerox = async ({
   openaiAPIKey = "",
   outputDir,
   tempDir = os.tmpdir(),
+  llmParams = {
```

Comment:
Since this is an object, what happens if a user passes in `llmParams` and does not include `maxTokens` but instead just `temperature`? Should we concat the two objects? We should also probably check to make sure that only the values that are allowed are passing into the object, otherwise throw an error.

Reply:
Added a validator